backends/rapidpro/task.go

```diff
@@ -16,9 +16,9 @@ func queueMsgHandling(rc redis.Conn, c *DBContact, m *DBMsg) error {
 	body := map[string]any{
 		"contact_id":      c.ID_,
 		"org_id":          channel.OrgID_,
-		"channel_id":      channel.ID_, // deprecated
-		"channel_uuid":    channel.UUID_,
-		"channel_type":    channel.ChannelType_,
+		"channel_id":      channel.ID_,
+		"channel_uuid":    channel.UUID_,        // deprecated
+		"channel_type":    channel.ChannelType_, // deprecated
```

Comment:
I was thinking mailroom would need these but the first thing mailroom does is grab that channel by its id and that then has the UUID and type

Reply:
Just so we remember reverting https://github.com/nyaruka/rapidpro/pull/4033/files as well

Reply:
Ayyyy I totally forgot well remembered!

Reply:
https://github.com/nyaruka/rapidpro/pull/4040